Assets/HoloToolkit/Input/Scripts/CursorManager.cs

```diff
@@ -24,32 +24,38 @@ public partial class CursorManager : Singleton<CursorManager>
 
     void Awake()
     {
-        if (CursorOnHolograms == null || CursorOffHolograms == null)
+        // Hide the Cursors to begin with.
+        if (CursorOnHolograms != null)
         {
-            return;
+            CursorOnHolograms.SetActive(false);
+        }
+        if (CursorOffHolograms != null)
+        {
+            CursorOffHolograms.SetActive(false);
         }
 
-        // Hide the Cursors to begin with.
-        CursorOnHolograms.SetActive(false);
-        CursorOffHolograms.SetActive(false);
+        // Make sure there is a GazeManager in the scene
+        if (FindObjectOfType<GazeManager>() == null)
+        {
+            Debug.LogWarning("CursorManager requires a GazeManager in your scene.");
+        }
```

Comment:
a consideration when we find these invalid configurations would be to Destroy(this) so the script doesn't run. This would remove the need to do null checks everywhere else.